src/routes/(auth)/login/+server.ts

```diff
@@ -27,9 +27,9 @@ export async function createToken(
 
   if (response?.error || response?.data?.tokenCreate?.error) {
     if (response?.data?.tokenCreate?.error) {
-      const error = response?.data?.tokenCreate?.error;
+      const error = response?.data?.tokenCreate;
```

Comment:
```suggestion
      const error = response.data.tokenCreate.error;
```

Based on if clause, theres no need for optional chaining here.

Also, you are not accessing the error in L30 neither.